spec/controllers/admin/features_controller_spec.rb

```diff
@@ -16,6 +15,65 @@
       get :show, :id => feature.id
       expect(response).to be_success
     end
-  end  
+  end
+
+  describe '#destroy' do
+    subject { delete :destroy, id: feature.id }
+
+    it 'redirects to the index action' do
+      expect(subject).to redirect_to action: :index
+    end
+
+    context 'when the feature can be deleted' do
+      before(:each) do
+        5.times do
+          FactoryGirl.create(:green_flag_rule, feature_id: feature.id)
+          FactoryGirl.create(:green_flag_feature_event, feature_id: feature.id)
+          FactoryGirl.create(:green_flag_feature_decision, feature_id: feature.id)
+        end
+
+        subject
+      end
+
+      it 'deletes the feature' do
+       expect(GreenFlag::Feature.find_by_id(feature.id)).to be_nil
+      end
+
+      it "deletes the feature's rules" do
+        expect(GreenFlag::Rule.where(feature_id: feature.id)).to eq []
+      end
+
+      it "deletes the feature's events" do
+        expect(GreenFlag::FeatureEvent.where(feature_id: feature.id)).to eq []
+      end
 
+      it "deletes the feature's feature decisions" do
+        expect(GreenFlag::FeatureDecision.where(feature_id: feature.id)).to eq []
+      end
+
+      it 'sets a successful flash notice' do
+        expect(flash[:notice]).to eq "Feature \"#{feature.code}\" has been successfully deleted."
+      end
+    end
+
+    context 'when the feature cannot be deleted' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.